### PR TITLE
Update CHANGELOG `1.106.0`

### PR DESCRIPTION
**Pull request by confused-Techie**
Generic CHANGELOG update of each PR

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,11 +6,38 @@
...
+## 1.106.0
+
 - Fixed bug that happens on some systems when trying to launch Pulsar using the Cinnamon desktop environment
 - Added a modern implementation of Tree-sitter grammars behind an experimental flag. Enable the “Use Modern Tree-Sitter Implementation” in the Core settings to try it out
 - Bugfix: fixed Clojure indentation on tree-sitter
 - Improved the Clojure language support by migrating it to tree-sitter and support block comments, quoting, and other advanced features on modern tree-sitter implementation
-- Added a modern implementation of Tree-sitter grammars behind an experimental flag. Enable the “Use Modern Tree-Sitter Implementation” in the Core settings to try it out.
+- Fixed a bug that could cause images to not appear the first time opening them
+- `autocomplete-css` Completions are now sorted in a way that may match what users expect
+- Added a "Logout" option in the `github` package
```
Comment (DeeDeeG) [inline CHANGELOG.md:17]:
```suggestion
- Added a "Log Out" menu item for the `github` package
```

Maybe to clarify this isn't a new feature just a new UI element?

File: packages/welcome/lib/changelog-view.js [hunk 2]
```diff
@@ -53,30 +53,29 @@ export default class ChangeLogView {
...
               </li>
               <li>
-                Fixed RPM installation failure when Atom was installed on the same machine
+                `autocomplete-css` Completions are now sorted in a way that may match what users expect
               </li>
               <li>
-                Added a new set of Package `activationHooks`, `...:uri-opened` lets a package activate when any URI
-                is opened within Pulsar, and `...:file-name-opened` lets a package activate when any specific filename
-                is opened within Pulsar.
+                Added a "Logout" option in the `github` package
```
Comment (DeeDeeG) [inline packages/welcome/lib/changelog-view.js:78]:
```suggestion
                Added a "Log Out" menu item for the `github` package
```

Same suggestion as above, this is just in the other file

**Review by DeeDeeG — APPROVED** [review]
The GitHub changes look good to me, I'm not super familiar with the other changes, TBH, I could take extra moment to verify the PRs match exactly what was merged since 1.105.0, but I basically trust it to be right. Wait a minute to merge if you want me to verify all the PRs match, other than that LGTM 👍

**Comment by confused-Techie** [comment]
Appreciate it, I'll go ahead and merge this one

**Pull request merged**